posthog/hogql_queries/web_analytics/stats_table.py

```diff
@@ -132,9 +136,26 @@ def calculate(self):
             modifiers=self.modifiers,
         )
 
+        def to_data(col_val, col_idx):
+            if col_idx == 0:  # breakdown_value
+                return col_val
+            elif col_idx == 1:  # views
+                return self._unsample(col_val)
+            elif col_idx == 2:  # visitors
+                return self._unsample(col_val)
+            elif col_idx == 3:  # bounce_rate
+                return col_val
+            else:
+                return col_val
+
+        if response.results:
```

Comment:
@robbie-c To make mypy happy, if you know that results will always be there, you can also put `assert response.results is not None` to narrow it and it will be happy.